src/pj_param.c

```diff
@@ -35,76 +93,76 @@ paralist *pj_mkparam(char *str) {
 /*                                                                      */
 /************************************************************************/
 
-	PROJVALUE /* test for presence or get parameter value */
+    PROJVALUE /* test for presence or get parameter value */
 pj_param(projCtx ctx, paralist *pl, const char *opt) {
 
-	int type;
-	unsigned l;
-	PROJVALUE value;
-
-	if( ctx == NULL )
-		ctx = pj_get_default_ctx();
-
-	type = *opt++;
-	/* simple linear lookup */
-	l = (int)strlen(opt);
-	while (pl && !(!strncmp(pl->param, opt, l) &&
-	  (!pl->param[l] || pl->param[l] == '=')))
-		pl = pl->next;
-	if (type == 't')
-		value.i = pl != 0;
-	else if (pl) {
-		pl->used |= 1;
-		opt = pl->param + l;
-		if (*opt == '=')
-			++opt;
-		switch (type) {
-		case 'i':	/* integer input */
-			value.i = atoi(opt);
-			break;
-		case 'd':	/* simple real input */
-			value.f = pj_atof(opt);
-			break;
-		case 'r':	/* degrees input */
-			value.f = dmstor_ctx(ctx, opt, 0);
-			break;
-		case 's':	/* char string */
+    int type;
```

Comment:
If this is all just formatting, can you commit this separately from this PR to reduce the noise in this pr?

Reply:
I could, and I am actually surprised to see this amount of formatting changes show up - this is definitlely a case of editor setup hiding some changes.

I do, however, think (although I may misremember, and @kbevers  a.o., please correct me if I'm wOrNg)  that about a year ago, after @kbevers  did a number of commits unifying whitespace (to the canonical Guido van Rossum compatible) "no tabs, 4 spaces per indent"), we agreeed that the remaining work on this would have to be "drive by" corrections. Unintendedly, this is one of them...